Assignment4/ralonsosirera-190182/task06.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2020-2021/master/Assignment4"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/resources/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**"""
+g.add((ns.University, RDF.type, RDFS.Class))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, VCARD.Given, Literal("Jane")))
+g.add((ns.JaneSmith, VCARD.Family, Literal("Smith")))
+# Visualize the results
+for s, p, o in g:
+    print(s,p,o)
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.worksAt, ns.UPM))
```

Comment:
property worksAt does not exist in vcard. Create your own

Reply:
Hello, is it okay if I solve all the problems with !pip, properties and hierarchy in 7 by opening a new pr?

Reply:
you may do as you prefer.